apsuite/commisslib/injbo_optimize_rcds.py

```diff
@@ -16,10 +16,53 @@ class OptimizeInjBOParams(_RCDSParams):
     """."""
 
     KNOBS = [
-        'li_qf3', 'tb_qf2a', 'tb_qf2b', 'tb_qd2a', 'tb_qd2b', 'posx', 'angx',
-        'posy', 'angy', 'kckr']
-    LIMS_UPPER = [+3.0, 9.5, 6.0, 8.5, 5.5, +2.0, +1.0, +2.0, +1.0, -25.0]
-    LIMS_LOWER = [-3.0, 5.0, 2.0, 4.0, 1.5, -2.0, -1.0, -2.0, -1.0, -19.0]
+        'li_qf3',
```

Comment:
@vellosok75 , I find this grouping of KNOB parameters unnatural and more difficult to read. I created a branch `OptimizeInjBOParams` with suggestions. please take a look at it and if you agree, merge it into this PR, if not delete the branch.